doc/node.1

```diff
@@ -2,7 +2,7 @@
 
 .\" This is a man page comment.
 .
-.\" Man page syntax (actually troff syntax) is somewhat obscure, but the
+.\" Man page syntax (actually roff syntax) is somewhat obscure, but the
```

Comment:
[Brief explanation](https://github.com/github/linguist/pull/3126#issuecomment-234164315) on the difference between troff and roff, dated 12 hours ago.